Index: html/docbook.xsl
===================================================================
--- html/docbook.xsl	(Revision 6824)
+++ html/docbook.xsl	(Arbeitskopie)
@@ -34,7 +34,7 @@
 <xsl:include href="metadata.xsl"/>
 
 <!-- Overwrite standard header and footer with our own definitions -->
-<xsl:include  href="navig.header.footer.xsl"/>
+<!-- <xsl:include  href="navig.header.footer.xsl"/> -->
 
 <!-- Overwrites some template rules for graphics -->
 <xsl:include href="graphics.xsl"/>
Index: html/sections.xsl
===================================================================
--- html/sections.xsl	(Revision 6824)
+++ html/sections.xsl	(Arbeitskopie)
@@ -50,7 +50,7 @@
 </xsl:template>
 
 
-<xsl:template name="section.heading" priority="100">
+<xsl:template name="section.heading">
   <xsl:param name="section" select="."/>
   <xsl:param name="level" select="1"/>
   <xsl:param name="allow-anchors" select="1"/>
Index: html/param.xsl
===================================================================
--- html/param.xsl	(Revision 6824)
+++ html/param.xsl	(Arbeitskopie)
@@ -99,5 +99,7 @@
 
 <!--  Output a warning, if chapter/@lang is different from book/@lang ? -->
 <xsl:param name="warn.xrefs.into.diff.lang" select="1"/>
-  
+
+<!-- -->
+<xsl:param name="show.remarks" select="0"/>  
 </xsl:stylesheet>
Index: html/xref.xsl
===================================================================
--- html/xref.xsl	(Revision 6824)
+++ html/xref.xsl	(Arbeitskopie)
@@ -39,8 +39,8 @@
       <xsl:choose>
         <xsl:when test="/set/@id=$rootid or
                         /article/@id=$rootid">
-           <!--<xsl:apply-imports/>--><!-- If we use the whole set, do the usual stuff -->
-           <xsl:call-template name="xref.old"/>
+           <xsl:apply-imports/><!-- If we use the whole set, do the usual stuff -->
+           <!-- <xsl:call-template name="xref.old"/> -->
         </xsl:when>
         <xsl:otherwise>
           <!-- A reference into another book -->
